src/handlers/localstorage/common.ts

```diff
@@ -62,13 +131,59 @@ export const getLocal = async (key = '', version = defaultVersion) => {
  * @desc get from storage
  * @param  {String}  [key='']
  * @return {Object}
+ *
+ */
+export const getLocal = async (key = '', version = defaultVersion) => {
+  try {
+    const mmkvData = await getLocalMMKV({ key, version });
+
+    // if we have data in mmkv we want to use that
+    if (mmkvData) {
+      return mmkvData;
+    } else {
+      // otherwise we want to get the data from the legacy async storage
```

Comment:
placeholder note: did we want to add it to the local MMKV and then remove the local async here